chat-plugins/info.js

```diff
@@ -636,6 +646,33 @@ var commands = exports.commands = {
 		"The parameter 'mega' can be added to search for Mega Evolutions only, and the parameters 'FE' or 'NFE' can be added to search fully or not-fully evolved Pokemon only.",
 		"The order of the parameters does not matter."],
 
+	randpoke: 'randompokemon',
+	randompokemon: function (target) {
+		var targets = target.split(",");
+		var targetsBuffer = [];
+		var qty;
+		for (var i = 0; i < targets.length; i++) {
+			if (!targets[i]) continue;
+			var num = Number(targets[i]);
+			if (Number.isInteger(num)) {
+				if (qty) return this.sendReply("Only specify the number of Pokemon once.");
+				qty = num;
+				if (1 > qty || qty > 15) return this.sendReply("Number of random Pokemon must be between 1 and 15.");
+				targetsBuffer.push("random" + qty);
+			} else {
+				targetsBuffer.push(targets[i]);
+			}
+		}
+		if (!qty) targetsBuffer.push("random1");
+		//Reassigns the target variable to be called in /dexsearch
+		var argumentsBuffer = arguments;
+		argumentsBuffer["0"] = targetsBuffer.join(",");
```

Comment:
hmm, does `this.parse` preserve the `cmd` variable being the original command? The reason I used `apply` is because I didn't want the random functionality to be possible to access with `/dexsearch` as that might create some confusion.